### PR TITLE
Added The Bifrost Testnet network and The Bifrost icon

### PR DESCRIPTION
**Pull request by mur-me**

### CODE REVIEW

File: _data/icons/thebifrost.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafkreid32bybgacpg7v3rrpq56oplhegmoyzhygzbvykkigv62f5a3iltu",
+    "width": 1000,
+    "height": 1000,
+    "format": "png"
```
Comment (ligi) [inline _data/icons/thebifrost.json:6]: the sizes are wrong
```
✓ ~/tmp identify bafkreid32bybgacpg7v3rrpq56oplhegmoyzhygzbvykkigv62f5a3iltu
bafkreid32bybgacpg7v3rrpq56oplhegmoyzhygzbvykkigv62f5a3iltu PNG 670x175 670x175+0+0 8-bit sRGB 18352B 0.000u 0:00.000
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/thebifrost.json (hunk 1 above)
Comment (mur-me) [reply _data/icons/thebifrost.json:6]: Thanks, one sec, I'll change!

File: _data/chains/eip155-49088.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://goerli.net/#about",
+  "shortName": "gor",
+  "chainId": 49088,
+  "networkId": 49088,
+  "explorers": [
+    {
+      "name": "explorer-thebifrost",
+      "url": "https://explorer.testnet.thebifrost.io/",
```
Comment (ligi) [inline _data/chains/eip155-49088.json:22]:
```suggestion
      "url": "https://explorer.testnet.thebifrost.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by mur-me** [comment]
Hi @ligi,

Is it normal that some of the GitHub actions jobs have failed after merge?

**Comment by ligi** [comment]
yes - you cannot use gor as a shortName - so I reverted your PR - please submit another one with a not yet used shortname

**Comment by mur-me** [comment]
@ligi sorry for my fuckup, I've fixed everything, check it twice and created new pr - https://github.com/ethereum-lists/chains/pull/1766
can you please take a look on it, if u have time?

**Pull request merged**